backend/backend/settings.py

```
@@ -0,0 +1,126 @@
+from dotenv import load_dotenv
+import os
+from urllib.parse import urlparse
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+SECRET_KEY = 'django-insecure-w#vc9rf7#xi*5$%z3^*z5gl@0y!e%qc6hhdr0ctg*veb5ca1^*'
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)